### PR TITLE
Foreman 3.0 release announcement

### PR DESCRIPTION
**Pull request by lzap**
SSIA

I spiced it up with the screenshots of the experimental UI

### CODE REVIEW

File: _posts/2021-09-16-foreman-30-is-here.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Foreman 3.0 is an important milestone on the road to modernizing Foreman’s user
+interface. In this release, our main focus is on the host detail page. The
+experimental and all new host details page is finally ready for users to start
+testing it and sending us feedback via our community forums or IRC.
+
+<img src="/static/images/blog_images/2021-09-16-new-foreman-host-page.png"  />
+
+To enable the new host page, head over to Administer - Settings - Generic and
+set Show Experimental Labs setting to Yes. At this point, clicking on a host
+detail will still open the current host detail page. To view the new page, on
```
Comment (melcorr) [inline _posts/2021-09-16-foreman-30-is-here.md]:
```suggestion
will still open the current host detail page. To view the new page, on
```

File: _posts/2021-09-16-foreman-30-is-here.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+We are looking for your feedback that will help us create the best possible user interface.
+
+<img src="/static/images/blog_images/2021-09-16-new-foreman-status-page.png"  />
+
+### Puppet ENC functionality extracted to foreman_puppet plugin
+
+Foreman without the plugin no longer includes Puppet environments, config
+groups, Puppet classes and smart class parameters. It is still possible to
+assign Puppet environment to a host using the `puppet_environment` parameter.
```
Comment (melcorr) [inline _posts/2021-09-16-foreman-30-is-here.md]:
```suggestion
assign a Puppet environment to a host using the `puppet_environment` parameter.
```

File: _posts/2021-09-16-foreman-30-is-here.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+upgraded to use `mod_auth_gssapi` too. This allows deploying Kerberos
+authentication on CentOS 8, which lacks `mod_auth_kerb`. Debian and Ubuntu also
+provide `mod_auth_gssapi` and the installer can now deploy Kerberos
+authentication there too.
+
+### Fact parsers are included in core
+
+Fact parsers for Ansible, Chef, Salt and Subscription Manager were moved to the
+Foreman core. This is the first step in refactoring fact handling in Foreman to
+be simpler, faster and easily extendable. Thanks to that it's now possible to
```
Comment (melcorr) [inline _posts/2021-09-16-foreman-30-is-here.md]:
```suggestion
be simpler, faster and easily extendable. Thanks to that, it's now possible to
```

**Review by melcorr — CHANGES_REQUESTED** [review]
A few small things but otherwise good to go.

**Comment by lzap** [comment]
TY commited all of them, please squash, thanks!

**Pull request merged**